package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "parsedbf",
-  "version": "0.1.1",
```

Comment:
I'll update this when I publish (also it's a bug fix so would be 0.1.2, but don't touch this I'll use npm version to deal with it)